dynparquet/schema.go

```diff
@@ -363,14 +367,21 @@ func (s *Schema) NewBuffer(dynamicColumns map[string][]string) (*Buffer, error)
 	}, nil
 }
 
+var rowBufPool = &sync.Pool{
+	New: func() interface{} {
+		return make([]parquet.Row, 64) // Random guess.
+	},
+}
+
```

Comment:
Does it make sense to have this initialized when calling `NewSchema` rather than having it globally in the package?